cli/src/commit_templater.rs

```diff
@@ -616,6 +625,11 @@ fn builtin_commit_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, Comm
             Ok(L::wrap_ref_name_list(out_property))
         },
     );
+    // TODO: Remove the following block after jj 0.28+
+    map.insert("branches", map["bookmarks"]);
+    map.insert("local_branches", map["local_bookmarks"]);
+    map.insert("remote_branches", map["remote_bookmarks"]);
```

Comment:
We'll probably need to duplicate color definitions in `cli/src/config/colors.toml` (with TODO comment.) Both `branch` and `bookmark` have to be colorized.

Reply:
done